### ISSUE TITLE
Update pytesseract, specify Tesseract version

### ISSUE THREAD

**Issue by travellingprog** [electricitymaps/electricitymaps-contrib#5933]
## Description

TL;DR pytesseract should be updated to a newer version, and the wiki should specify which Tesseract version to install.

Hi, I just started looking into this project, and I started following the instructions for setting up the local environment, from the wiki. One of the instructions is to install Tesseract, to which I asked myself "which version of Tesseract?". In my Linux Mint distro, for example, the default tesseract-ocr package points to a v4 release, but v5 has been out for a while and requires adding a PPA.

I did a search in the repo files to see if a version was specified anywhere, and I saw that this project is using pytesseract v0.2.0. I looked at [the release notes for that version](https://github.com/madmaze/pytesseract/releases/tag/v0.2.0) and saw that it came out on January 30 2018. I compared that date with the Tesseract releases, and saw that the stable release at that time was v3.05.01.

I imagine that most developers that are new to this project are installing Tesseract v4 or v5, rather than v3. This will likely cause an issue at some point. So there should be a decision made on which Tesseract release to use, and pytesseract should be updated to the latest version compatible with that release.

If someone can comment on which Tesseract version to use, I could create the PR myself.

**Comment by VIKTORVAV99** [comment]
We are closing down some longstanding issues that have't had any activity in a while. If you believe this is still an issue let us know and we can reopen it.
